server/sonar-web/src/main/js/apps/organizations/navigation/OrganizationNavigation.tsx

```diff
@@ -23,93 +23,72 @@ import OrganizationNavigationMenu from './OrganizationNavigationMenu';
-      } else {
-        this.setState({ error: '' });  
-      }
-    }).catch(throwGlobalError)
-  }
-
+export function OrganizationNavigation({ location, organization, userOrganizations }: Props) {
+  const [notifications, setNotifications] = React.useState<Notification[]>([]);
+  const { contextNavHeightRaw, contextNavHeightWithError } = rawSizes;
+  const [height, setHeight] = React.useState(contextNavHeightRaw);
+  const orgAlertHeight = 30; // refer .org-alert-warning height in NavBarTabs
 
-  componentWillUnmount() {
-    this.mounted = false;
-  }
+  React.useEffect(() => {
+    const fetchNotifications = async () => {
+        const notifications = await getRawNotificationsForOrganization(organization.kee);
+        if(notifications.length > 0) {
+          const errorNotifications = notifications.filter(notification => notification.type == 'ERROR');
+          if(errorNotifications.length > 0) {
+            setNotifications(errorNotifications);
+            setHeight(contextNavHeightWithError + orgAlertHeight * (errorNotifications.length - 1));
+          } else {
+            setNotifications(notifications);
+            setHeight(contextNavHeightWithError + orgAlertHeight * (notifications.length - 1));
+          }
+        } else {
+          setNotifications([]);
+          setHeight(contextNavHeightRaw);
+        }
+    };
+    fetchNotifications();
+  },[organization, userOrganizations]);
 
-  render(){
-    const { contextNavHeightRaw, contextNavHeightWithError } = rawSizes;
-    const {location,organization,userOrganizations} = {...this.props}  
-    const {error} = {...this.state};
-    
-    const height = (error?.length>0) ? contextNavHeightWithError : contextNavHeightRaw;
-    return (
-      <>
-      <ContextNavBar height={height} id="context-navigation">
-        <div className="navbar-context-justified">
-          <OrganizationNavigationHeader
-              organization={organization}
-              organizations={userOrganizations}
-          />
-          <OrganizationNavigationMeta
+  return (
+        <>
+        <ContextNavBar height={height} id="context-navigation">
+          <div className="navbar-context-justified">
+            <OrganizationNavigationHeader
+                organization={organization}
+                organizations={userOrganizations}
+            />
+            <OrganizationNavigationMeta
+                organization={organization}
+            />
+          </div>
+          <OrganizationNavigationMenu
+              location={location}
               organization={organization}
           />
-        </div>
-        <OrganizationNavigationMenu
-            location={location}
-            organization={organization}
-        />
-        {
-        error?.length>0 ? (
-        <div className='org-alert'>
-          <div className='org-alert-inner'>
-            <div className='icon'>
-              x
+          { notifications.length>0 ? notifications.map((notification, key) => (
```

Comment:
I believe there is no need in the if condition here:
`notifications.length>0`

You can simply use 

```
{ notifications.map((notification, key) => (
<div className={notification.type === 'ERROR' ? 'org-alert-error' : 'org-alert-warning'} key = {key}>
              <div className='org-alert-inner'>
                <div className='icon'>
                  {notification.type === 'ERROR' ? 'x' : '!'}
                </div>
                <div className='msg' dangerouslySetInnerHTML={{ __html: sanitizeUserInput(notification.message) }} />
              </div>
            </div>
          ))
     }
```